django/core/static/css/custom.css

```diff
@@ -106,28 +106,32 @@ main a {
 
 /* ---------- Welcome ---------- */
 
-#welcome-videobanner {
-    max-height: 40em;
-    overflow: hidden;
+#welcome-banner {
+    min-height: 75vh;
+    background-image: url('../images/welcome/main-banner.jpg');
+    background-size: cover;
+	background-position: center;
```

Comment:
Very minor issue of spacing for lines 113 - 115

Reply:
thanks for spotting. causes by the dreaded tabs instead of spaces. all sorted now, cheers